doc/source/user_guide/groupby.rst

```diff
@@ -799,7 +799,7 @@ Another common data transform is to replace missing data with the group mean.
    transformed = grouped.transform(lambda x: x.fillna(x.mean()))
 
 We can verify that the group means have not changed in the transformed data
-and that the transformed data contains no NAs.
+and that the transformed data contains no NAN's.
```

Comment:
We try to use NA to refer to missing values in generally. We'll sometimes use `np.nan` when talking about that specific specific missing value.

Reply:
Ok, got it!